community.md

```diff
@@ -55,6 +55,7 @@ It would be great to hear from you especially if you have any of the above and w
 | [Hands-on Serverless with OpenFaaS and Python](https://gotochgo.com/2019/workshops/152) | Alex Ellis | Chicago, USA | 1-May-2019 |
 | [GOTO Chicago - Serverless Beyond the Hype](https://gotochgo.com/2019/sessions/728) | Alex Ellis | Chicago, USA | 30-Apr-2019 |
 | [Serverless Architecture Conf: OpenFaaS - Keeping Serverless Simple](https://serverless-architecture.io/serverless-platforms-technology) | John McCabe | The Hague, Netherlands | 9-Apr-2019 |
+| [openfaas in the vmware ecosystem - webinar](https://www.terasky.com/news/openfaas-in-the-vmware-ecosystem-webinar-april-3rd-2019/) | Scott Rosenberg | online from Israel | 3-Apr-2019 |
```

Comment:
Can we be sensitive to branding:

openfaas => OpenFaaS
vmware => VMware

Reply:
updated in new commit